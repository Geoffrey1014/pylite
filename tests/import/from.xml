lambda
  name __init__
  param_count 0
  local_count 0
  label_count 0
  insn_vector
    load_module c0
    load_field c1
    load_this_module
    store_field c1
    load_module c0
    load_field c2
    load_this_module
    store_field c2
    load_global c1
    print
    load_const c3
    load_const c4
    load_global c2
    call 2
    print
    load_const c5
    return
  const_vector
    math
    pi
    pow
    10.000000
    2.000000
    None
  guard_vector
  line_number_map 1 1 1 1 1 1 1 1 3 3 4 4 4 4 4 1 1
