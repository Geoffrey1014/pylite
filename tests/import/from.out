3.14159
1024
